ironfish/src/strategy.ts

```diff
@@ -76,12 +76,16 @@ export class Strategy {
 
     const annualReward = (GENESIS_SUPPLY_IN_IRON / 4) * Math.E ** (-0.05 * yearsAfterLaunch)
 
-    reward = this.convertIronToOre(annualReward / IRON_FISH_YEAR_IN_BLOCKS)
+    reward = this.convertIronToOre(this.mRound(annualReward / IRON_FISH_YEAR_IN_BLOCKS, 0.125))
     this.miningRewardCachedByYear.set(yearsAfterLaunch, reward)
 
     return reward
   }
 
+  mRound(num: number, threshold: number): number {
```

Comment:
What do you think about moving this here next to this, and giving it a better name? It seems useful. https://github.com/iron-fish/ironfish/blob/master/ironfish/src/utils/math.ts#L29-L32

Reply:
Fixed